crates/matrix-sdk-store-encryption/src/lib.rs

```diff
@@ -146,12 +181,59 @@ impl StoreCipher {
 
         keys.zeroize();
 
-        let store_cipher = EncryptedStoreCipher {
-            kdf_info: KdfInfo::Pbkdf2ToChaCha20Poly1305 { rounds: kdf_rounds, kdf_salt: salt },
+        Ok(EncryptedStoreCipher {
+            kdf_info,
             ciphertext_info: CipherTextInfo::ChaCha20Poly1305 { nonce, ciphertext },
+        })
+    }
+
+    #[doc(hidden)]
+    pub fn _insecure_export_fast_for_testing(&self, passphrase: &str) -> Result<Vec<u8>, Error> {
+        self.export_kdf(passphrase, 1000)
+    }
```

Comment:
Can we gate this behind the testing feature?

Reply:
I guess we can, but does this give us something, nobody will use this by accident, nobody will discover it accidentally either since it's hidden from the docs.

If you feel strongly about this we can do this in a separate PR, this one has been waiting for a while and adding a feature flag is non-trivial considering our dependency tree.